features/download_item_list.feature

```diff
@@ -47,3 +41,25 @@ Feature: Managing Item Lists
     And I should see "Download Selected"
     And I should see "Download only files that match a particular regular expression"
     And I should see "Download Matches"
+
+
+  Scenario: Downloading more than 200 documents results in an error
+    Given "researcher@intersect.org.au" has item lists
+      | name  |
+      | Test1 |
+    And the item list "Test1" has 101 remote documents
```

Comment:
Perhaps this should use "/^the item list "(.*?)" has (\d+) text documents$/" rather than the remote documents, as remote documents seem to be handled by the next test below (Remote documents can be downloaded not result in an error).